pandas/core/computation/eval.py

```diff
@@ -212,7 +212,7 @@ def eval(
 
     truediv : bool, optional
         Whether to use true division, like in Python >= 3.
-        deprecated:: 1.0.0
+        deprecated:: 1.0.0.
```

Comment:
Thanks @Iqrar99 

The period should not be added here.

There should be a blank line before this to ensure it renders correctly. see https://pandas.pydata.org/pandas-docs/dev/reference/api/pandas.eval.html?highlight=deprecated

 if you remove the period and add the blank line, does that resolve the PR09 error.